src/main/resources/com/linecorp/armeria/server/docs/assets/js/armeria.js

```diff
@@ -189,6 +206,20 @@ $(function () {
                  });
   }
 
+  function escapeHtml(string) {
```

Comment:
Maybe it's time to add the "javascript standard library"? ;)

https://lodash.com/docs#escape

No worries either way